### ISSUE TITLE
Can you support swig?

### ISSUE THREAD

**Issue by guohuihot** [gulp-sourcemaps/gulp-sourcemaps#315]
Can you support swig?
[gulp-swig](https://github.com/colynb/gulp-swig)

**Comment by phated** [comment]
Plugin support is controlled by the author of that plugin.

**Comment by guohuihot** [comment]
But after I used gulp-swig, the number of lines was wrong

**Comment by guohuihot** [comment]
![image](https://user-images.githubusercontent.com/3119995/26857459-451250ee-4b5d-11e7-8ee3-6b22e2aae2b1.png)
![image](https://user-images.githubusercontent.com/3119995/26857474-5637c66a-4b5d-11e7-8df4-8c2bae3c5002.png)
The prompt is on the 66 line, but actually the 62 line

**Comment by nmccready** [comment]
More than likely this is an issue with gulp-swig. But you are missing some important facts to back up your claim.  Where you are missing a reproducible MVCE. 

tldr: We Need Small example that makes your problem happen.

Anyway if you keep responding with little to no detail the issue will be locked.

**Comment by guohuihot** [comment]
![image](https://user-images.githubusercontent.com/3119995/26860826-9f3c1586-4b72-11e7-84ad-fd37ded9fe8d.png)
Because the top adds a line of swig code, if I remove it, that's right;
That's why I asked if I supported gulp-swig

**Comment by nmccready** [comment]
Locking; all I am asking is for you to create a gulp pipeline for swig.
